src/view/show_mode/editor/show_ui_widgets/cue_control.py

```diff
@@ -71,21 +93,41 @@ def construct_widget(self, parent: QWidget | None, enabled: bool):
             item.annotated_data = cue
             cue_list.addItem(item)
         cue_list.setEnabled(enabled)
-        cue_list.setMinimumHeight(100)
+        cue_list.setMinimumHeight(300)
         if enabled:
             self._player_cue_list_widget = cue_list
         else:
             self._config_cue_list_widget = cue_list
         layout.addWidget(cue_list)
+
+        self._progressBar.setParent(w)
+        self._progressBar.setEnabled(enabled)
+        self._progressBar.setMinimumHeight(60)
+        self._progressBar.setMinimum(0)
+        self._progressBar.setMaximum(20)
+        layout.addWidget(self._progressBar)
+
+        self._statuslabel.setParent(w)
+        self._statuslabel.setEnabled(enabled)
+        self._statuslabel.setMinimumHeight(40)
+        self._statuslabel.setVisible(True)
+        self._statuslabel.setText("Init text")
+        self._statuslabel.show()
+        layout.addWidget(self._statuslabel)
+
+        self.update_time_passed()
+
         w.setLayout(layout)
         return w
 
     def insert_action(self, action: str | None, state: str | None):
+        print("inact: " + action + " ")
         if not action or not state:
             return
         command = (action, state)
         self._command_chain.append(command)
         self.push_update()
+        self._command_chain.clear()
```

Comment:
here was a bug, when should the _command_chain should get cleared, after the commands were sent to fish?

Reply:
It should get cleared after `push_update`

Reply:
so like it is now?

Reply:
Yes.